### DISCUSSION TITLE
Is adding a samba share from NAS as a storage drive possible ?

### DISCUSSION THREAD

**Discussion by SamD** [IceWhaleTech/CasaOS#1303] [Q&A]
For background today was first install of CasaOS setup on Proxmox node, I'm new to both 

I see I am able to add the NAS share as network storage in the Files app/browser; however, I can't seem to be able to add this to the storage list itself. The only thing that lists is the main internal storage

Is this possible ?

**Comment by sesquivels** [comment]
Hello Sam, 
Thank you for started this thread, I have a question, about "able to add this storage list itself, something like the following picture?
![Screenshot 2023-07-31 155047](https://github.com/IceWhaleTech/CasaOS/assets/40394148/4af1087a-d324-4882-a896-f2e0bdb1c63e)
"
Please let me know in order to assist you properly,

sesquivels.

**Reply by SamD** [reply]
Yes I can add it here but I am unable to the Files but not the storage, perhaps this intentional due to performance over the network being slow
